<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingClass</class>
 <widget class="QMainWindow" name="ImageProcessingClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1070</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1070</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像处理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImageProcessing.qrc">
    <normaloff>:/ImageProcessing/image/icon.png</normaloff>:/ImageProcessing/image/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);

gridline-color: rgb(0, 0, 0);


color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>460</width>
      <height>460</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>源图像</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>560</width>
      <height>581</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>1图</string>
     </attribute>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>460</width>
        <height>460</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>480</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>处理图像</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>还原</string>
     </attribute>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>460</width>
        <height>460</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>label1</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDoc">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionRestore"/>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionCopyright"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuProc">
    <property name="title">
     <string>翻转</string>
    </property>
    <addaction name="Horizen"/>
    <addaction name="vertical"/>
    <addaction name="horizenAndVertical"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>形态</string>
    </property>
    <addaction name="Erosion"/>
    <addaction name="dilation"/>
    <addaction name="action1_19"/>
    <addaction name="action1_20"/>
    <addaction name="action1_21"/>
    <addaction name="action1_22"/>
    <addaction name="action1_23"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>滤波</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>缩放</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>边缘特征</string>
    </property>
    <addaction name="actionSobel"/>
    <addaction name="actionCanny"/>
    <addaction name="actionLaplacian"/>
    <addaction name="actionScharr"/>
   </widget>
   <widget class="QMenu" name="menuSobel">
    <property name="title">
     <string>Sobel</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>方向</string>
     </property>
     <addaction name="actionDx"/>
     <addaction name="actionDy"/>
     <addaction name="actionDxDy"/>
    </widget>
    <widget class="QMenu" name="menuKsize">
     <property name="title">
      <string>Ksize大小</string>
     </property>
     <addaction name="actionsize1"/>
     <addaction name="actionsize3"/>
     <addaction name="actionsize5"/>
     <addaction name="actionsize7"/>
    </widget>
    <widget class="QMenu" name="menuscale">
     <property name="title">
      <string>scale缩放因子</string>
     </property>
     <addaction name="actionscale05"/>
     <addaction name="actionscale1"/>
     <addaction name="actionscale2"/>
    </widget>
    <widget class="QMenu" name="menudelta">
     <property name="title">
      <string>delta值</string>
     </property>
     <addaction name="actiondelta0"/>
     <addaction name="actiondelta50"/>
     <addaction name="actiondelta100"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menuKsize"/>
    <addaction name="menuscale"/>
    <addaction name="menudelta"/>
   </widget>
   <addaction name="menuDoc"/>
   <addaction name="menuProc"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu"/>
   <addaction name="menuSobel"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionCopyright">
   <property name="text">
    <string>版权</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>还原</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionOpenTest">
   <property name="text">
    <string>打开测试文件</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>滤波</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>缩放</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="action1_6">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_7">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_8">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_9">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_10">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_11">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_12">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_13">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="Horizen">
   <property name="text">
    <string>水平</string>
   </property>
  </action>
  <action name="vertical">
   <property name="text">
    <string>垂直</string>
   </property>
  </action>
  <action name="horizenAndVertical">
   <property name="text">
    <string>水平垂直</string>
   </property>
  </action>
  <action name="Erosion">
   <property name="text">
    <string>腐蚀</string>
   </property>
  </action>
  <action name="dilation">
   <property name="text">
    <string>膨胀</string>
   </property>
  </action>
  <action name="action1_19">
   <property name="text">
    <string>开运算</string>
   </property>
  </action>
  <action name="action1_20">
   <property name="text">
    <string>闭运算</string>
   </property>
  </action>
  <action name="action1_21">
   <property name="text">
    <string>Morphological Gradient</string>
   </property>
  </action>
  <action name="action1_22">
   <property name="text">
    <string>顶帽</string>
   </property>
  </action>
  <action name="action1_23">
   <property name="text">
    <string>黑帽</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionLaplacian">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionScharr">
   <property name="text">
    <string>Scharr</string>
   </property>
  </action>
  <action name="actionDx">
   <property name="text">
    <string>dx</string>
   </property>
  </action>
  <action name="actionDy">
   <property name="text">
    <string>dy</string>
   </property>
  </action>
  <action name="actionDxDy">
   <property name="text">
    <string>dx&amp;dy</string>
   </property>
   <property name="iconText">
    <string>dx&amp;dy</string>
   </property>
   <property name="toolTip">
    <string>dx&amp;dy</string>
   </property>
  </action>
  <action name="actionsize1">
   <property name="text">
    <string>size=1</string>
   </property>
  </action>
  <action name="actionsize3">
   <property name="text">
    <string>size=3</string>
   </property>
  </action>
  <action name="actionsize5">
   <property name="text">
    <string>size=5</string>
   </property>
  </action>
  <action name="actionsize7">
   <property name="text">
    <string>size=7</string>
   </property>
  </action>
  <action name="actionscale05">
   <property name="text">
    <string>scale=0.5</string>
   </property>
  </action>
  <action name="actionscale1">
   <property name="text">
    <string>scale=1</string>
   </property>
  </action>
  <action name="actionscale2">
   <property name="text">
    <string>scale=2</string>
   </property>
  </action>
  <action name="actiondelta0">
   <property name="text">
    <string>delta=0</string>
   </property>
  </action>
  <action name="actiondelta50">
   <property name="text">
    <string>delta=50</string>
   </property>
  </action>
  <action name="actiondelta100">
   <property name="text">
    <string>delta=100</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageProcessing.qrc"/>
 </resources>
 <connections/>
</ui>
